src/pages/Coin/Tabs/InfoTab.tsx

```diff
@@ -30,18 +37,64 @@ export default function InfoTab({struct, data, supply}: InfoTabProps) {
 
   return (
     <Box marginBottom={3}>
-      <ContentBox>
-        <ContentRow title={"Name:"} value={data.data.name} />
-        <ContentRow title={"Symbol:"} value={data.data.symbol} />
-        <ContentRow title={"Decimals:"} value={data.data.decimals.toString()} />
-        <ContentRow title={"Total supply:"} value={formattedSupply} />
-        <ContentRow
-          title={"Creator:"}
-          value={
-            <HashButton hash={struct.split("::")[0]} type={HashType.ACCOUNT} />
-          }
-        />
-      </ContentBox>
+      {data && (
+        <ContentBox>
+          <ContentRow
+            title={"Name:"}
+            value={
+              <Typography
+                sx={{
+                  display: "flex",
+                  fontSize: "inherit",
+                  alignItems: "row",
+                  gap: 1,
+                }}
+              >
+                <span>{data?.data?.name}</span>
+                <VerifiedCoinCell
+                  data={{
+                    id: struct,
+                    known: !!coinData,
+                    isBanned: coinData?.isBanned,
+                    isInPanoraTokenList: coinData?.isInPanoraTokenList,
+                  }}
+                />
```

Comment:
Use that same component `Verified` (whatever) that we create here as well to avoid the `td`.